.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "**Required Info:**"
+  - type: input
+    id: os
+    attributes:
+      label: "Operating System:"
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: "Version or commit hash:"
+    validations:
+      required: true
+  - type: input      
+    id: dds
+    attributes:
+      label: "DDS implementation:"
+    validations:
+      required: true
```

Comment:
It does not have to be pull down, but can we add possible choices like, `rmw_fastrtps`, `rmw_connextdds` and `rmw_cyclonedds`? (there are more rmw implementations.) it would be easier for user to understand what needs to be filled in here especially beginners?

the same comment goes to `clientlib`.

Reply:
Sure, I was thinking about it.
The possibilities:
1) Placeholder text
2) Some explanation before the label. Examples and how the user can check it, like: "ros2 doctor --report and check the line: middleware name".
3) Dropdown with the option "Other". In this case, we need another field for input for "Other".

In all the cases, I may keep the explanation.
You can choose, what you prefer. Or you can suggest another method.
Péter